imports/packages/telegram/telegram.tsx

```diff
@@ -0,0 +1,301 @@
+
+    const { data: [{ id: telegramPackageLinkId }] } = await deep.insert({
+        type_id: packageTypeLinkId,
+        string: { data: { value: "@deep-foundation/telegram" } },
+        in: {
+            data: {
+                type_id: containTypeLinkId,
+                from_id: deep.linkId,
+            },
+        },
+    });
+
+    const { data: [{ id: tokenBotLinkId }] } = await deep.insert({
+        type_id: typeTypeLinkId,
+        in: {
+            data: {
+                type_id: containTypeLinkId,
+                from_id: telegramPackageLinkId,
+                string: { data: { value: "tokenBot" } },
+            },
+        },
+    });
+
+    const { data: [{ id: chatIdLinkId }] } = await deep.insert({
+        type_id: typeTypeLinkId,
+        in: {
+            data: {
+                type_id: containTypeLinkId,
+                from_id: telegramPackageLinkId,
+                string: { data: { value: "chatId" } },
+            },
+        },
+    });
+
+    const { data: [{ id: messageLinkId }] } = await deep.insert({
+        type_id: typeTypeLinkId,
+        in: {
+            data: {
+                type_id: containTypeLinkId,
+                from_id: telegramPackageLinkId,
+                string: { data: { value: "message" } },
+            },
+        },
+    });
+
+    const { data: [{ id: sendMessageLinkId }] } = await deep.insert({
+        type_id: typeTypeLinkId,
+        from_id: messageLinkId,
+        to_id: chatIdLinkId,
+        in: {
+            data: {
+                type_id: containTypeLinkId,
+                from_id: telegramPackageLinkId,
+                string: { data: { value: "sendMessage" } },
+            },
+        },
+    });
+
+    const { data: [{ id: hostLinkId }] } = await deep.insert({
+        type_id: typeTypeLinkId,
+        in: {
+            data: {
+                type_id: containTypeLinkId,
+                from_id: telegramPackageLinkId,
+                string: { data: { value: "host" } },
+            },
+        },
+    });
+
+    const { data: [{ id: setWebhookLinkId }] } = await deep.insert({
+        type_id: typeTypeLinkId,
+        from_id: tokenBotLinkId,
+        to_id: hostLinkId,
+        in: {
+            data: {
+                type_id: containTypeLinkId,
+                from_id: telegramPackageLinkId,
+                string: { data: { value: "setWebhook" } },
+            },
+        },
+    });
+
+    const handlerSendMessage = await insertHandler(
+        deep,
+        handleInsertTypeId,
+        sendMessageLinkId,
+        /* javascript */`
+            async ({deep, require, data: {newLink: {id: sendMessageLinkId}}}) => {
+                const {data: [{from_id: messageLinkId, to_id: chatIdLinkId}]} = await deep.select({
+                    id: sendMessageLinkId,
+                })
+                const {data: [{value:{value: messageValue}}]} = await deep.select({
```

Comment:
Check if link exists. [Example](https://github.com/deep-foundation/sdk/blob/1869110dfe1c3a141b8bd644e1c520b1dafaf147/imports/push-notification/notifyInsertHandler.js#LL11-L24C6)
This comment applies to all the places working like these lines of code